identity-service-logs/utils.js

```diff
@@ -214,6 +214,10 @@ async function getSelfUser() {
     });
 
     const self_user = await res.json();
+    if (self_user?.statusCode !== 200) {
+      alert('You are not loggedin. Please login!');
+      window.location.href = '/index.html';
+    }
     return self_user;
   } catch (err) {
```

Comment:
f the API call is successful (status code 200), the function returns the entire self_user object. However, it's not clear if the entire object is needed in all cases. Consider returning only the necessary information to avoid unnecessary data transfer.

Reply:
fixed it in new commit